### DISCUSSION TITLE
disable SSL chain completion in Nginx+ ingress

### DISCUSSION THREAD

**Discussion by Wonderingkaho** [nginx/kubernetes-ingress#2824] [Q&A]
Issue:

I saw k8s Nginx OSS ingress have an argument “--enable-ssl-chain-completion”, so is there have similar command argument in F5 Nginx + ingress? Or how can I disable it?

Thx

**Comment by pdabelf5** [comment]
The `ssl-chain-completion` feature is not available in F5 NGINX Ingress Controller.  If it is a feature you are looking for, please create a feature request issue.
